lib/solr_toggle/solr_toggle.rb

```diff
@@ -0,0 +1,18 @@
+module SolrToggle
+
+  def shouldIndexSolr
```

Comment:
Actually maybe I should rename this to `solrIsAvailable` and we could use it elsewhere to, for example, run searches in the API on Solr vs. on ActiveRecord... @icarito what do you think?